src/components/tooltip/tooltip.js

```diff
@@ -116,8 +116,15 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $mdUtil, $mdThe
 
     function bindEvents () {
       var autohide = scope.hasOwnProperty('autohide') ? scope.autohide : attr.hasOwnProperty('mdAutohide');
+      var clicked = false;
+      parent.on('click', function() { clicked = true; });
```

Comment:
I think you might actually need to bind to `mousedown`, as the enter key also triggers `click`.